packages/dnb-eufemia/src/extensions/forms/Field/BankAccountNumber/BankAccountNumber.tsx

```diff
@@ -1,22 +1,60 @@
-import React, { useMemo } from 'react'
+import React, { useCallback, useMemo } from 'react'
 import StringField, { Props as StringFieldProps } from '../String'
 
 import useErrorMessage from '../../hooks/useErrorMessage'
 import useTranslation from '../../hooks/useTranslation'
+import { Validator } from '../../types'
 
-export type Props = StringFieldProps & {
+export type Props = Omit<StringFieldProps, 'onBlurValidator'> & {
   validate?: boolean
   omitMask?: boolean
+  onBlurValidator?: Validator<string> | false
 }
 
 function BankAccountNumber(props: Props) {
   const translations = useTranslation().BankAccountNumber
```

Comment:
```suggestion
  const {
    errorBankAccountNumber,
    errorBankAccountNumberLength,
    errorRequired,
    label,
  } = useTranslation().BankAccountNumber
```

Just a nitpicky thought, that should not change functionality at all 😄 but could we remove the restructuring on line 17 to 22 and move it up to the useTranslation hook line? 🤔

Reply:
Done done.